src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -543,45 +530,6 @@ public void postAuthnLogin(String userAgent, String xForwardedFor,//NOSONAR
     }
   }
 
-  @Override
-  public void getAuthnCredentials(int length, int start, String sortBy, String query,
-      Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
-      Context vertxContext) {
-    try {
-       vertxContext.runOnContext(v -> {
-         String tenantId = getTenant(okapiHeaders);
-         String[] fieldList = {"*"};
-         try {
-           CQLWrapper cql = getCQL(query, length, start - 1);
-           PostgresClient.getInstance(vertxContext.owner(), tenantId).get(
-                   TABLE_NAME_CREDENTIALS, Credential.class, fieldList, cql, true, false, getReply -> {
-             if(getReply.failed()) {
-               logger.debug(POSTGRES_ERROR_GET + getReply.cause().getLocalizedMessage());
-               asyncResultHandler.handle(Future.succeededFuture(GetAuthnCredentialsResponse.respond500WithTextPlain(INTERNAL_ERROR)));
-             } else {
-               CredentialsListObject credentialsListObject = new CredentialsListObject();
-               List<Credential> credentialList = getReply.result().getResults();
-               credentialsListObject.setCredentials(credentialList);
-               credentialsListObject.setTotalRecords(getReply.result().getResultInfo().getTotalRecords());
-               asyncResultHandler.handle(Future.succeededFuture(GetAuthnCredentialsResponse.respond200WithApplicationJson(credentialsListObject)));
-             }
-           });
-         } catch(Exception e) {
-           logger.debug("Error invoking Postgresclient: "+ e.getLocalizedMessage());
-           asyncResultHandler.handle(Future.succeededFuture(GetAuthnCredentialsResponse.respond500WithTextPlain(INTERNAL_ERROR)));
-         }
-       });
-    } catch(Exception e) {
-      logger.debug(VERTX_CONTEXT_ERROR + e.getLocalizedMessage());
-      if(e.getCause() != null && e.getCause().getClass().getSimpleName().contains("CQLParseException")) {
-        asyncResultHandler.handle(Future.succeededFuture(GetAuthnCredentialsResponse.respond400WithTextPlain("CQL Parsing Error for '" + query + "': " +
-                e.getLocalizedMessage())));
-      } else {
-        asyncResultHandler.handle(Future.succeededFuture(GetAuthnCredentialsResponse.respond500WithTextPlain(INTERNAL_ERROR)));
-      }
-    }
-  }
-
```

Comment:
A test should be added that shows that the endpoint does not exist. The test should have a javadoc comment that points to https://issues.folio.org/browse/MODLOGIN-128 .